MyHeartCounts/Onboarding/PreferredWorkoutStep.swift

```diff
@@ -0,0 +1,114 @@
+//
+// This source file is part of the My Heart Counts iOS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import Foundation
+import SFSafeSymbols
+import SpeziOnboarding
+import SpeziViews
+import SwiftUI
+
+
+struct PreferredWorkoutStep: View {
+    struct NotificationTime: Hashable, Codable, Sendable {
+        var hour: Int
+        var minute: Int = 0
+    }
+    
+    @Environment(\.calendar)
+    private var calendar
+    @Environment(ManagedNavigationStack.Path.self)
+    private var path
+    
+    @State private var viewState: ViewState = .idle
+    @State private var workoutType: WorkoutType?
+    @State private var notificationTime = NotificationTime(hour: 9)
+    
+    var body: some View {
+        OnboardingPage(title: "Workout Preference", description: "WORKOUT_PREFERENCE_TEXT") {
+            selectionForm
+            DatePicker(
+                "Notification Time",
+                selection: Binding<Date> {
+                    calendar.date(
+                        bySettingHour: notificationTime.hour,
+                        minute: notificationTime.minute,
+                        second: 0,
+                        of: .now
+                    )! // swiftlint:disable:this force_unwrapping
+                } set: { newValue in
+                    let components = calendar.dateComponents([.hour, .minute], from: newValue)
+                    notificationTime = NotificationTime(
+                        hour: components.hour ?? 9,
+                        minute: components.minute ?? 0
+                    )
+                },
+                displayedComponents: [.hourAndMinute]
+            )
+        } footer: {
+            OnboardingActionsView("Continue", viewState: $viewState) {
+                // TODO: save the response somewhere!
```

Comment:
@PaulGoldschmidt Best place would probably be an other account key, ideally we could extract this easily attached to an account after the fact.